### DISCUSSION TITLE
[ FF8 ] Not responding after installing FFNx 1.15.0

### DISCUSSION THREAD

**Discussion by litnomad** [julianxhokaxhiu/FFNx#556] [Q&A] [answered]
**PLEASE NOTE:** If you come here to report a crash while you were using 7th Heaven, please do on their relative support channels. NO SUPPORT will be provided, and your issue will be close immediately.

**Describe the bug**
The FFNx startup screen slows and freezes when trying to run FFVIII

**To Reproduce**

I opened Steam and after hitting the play button on Final Fantasy VIII, another window opens that says FFNx but it's not responding.

**Expected behavior**
I was trying to play Final Fantasy VIII after installing FFNx according to https://steamcommunity.com/sharedfiles/filedetails/?id=2724995162 for music mods.

**Screenshots**

![Capture](https://user-images.githubusercontent.com/128262375/226143358-9c4aae35-d452-4289-a527-8c1959f40446.PNG)

**GPU (please complete the following information):**

- Brand = NVIDIA 
- Processor = Intel(R) Core(TM) i5-6200U CPU @ 2.30GHz   2.40 GHz
- Driver Version ?

**Additional context**

**Comment by julianxhokaxhiu** [comment, marked as answer]
Please use [the latest canary builds](https://github.com/julianxhokaxhiu/FFNx/releases/tag/canary) instead of stable for FF8, as there was a bug on the stable release that won't allow you to run the game properly.

**Comment by litnomad** [comment]
I just fixed it by downloading an older version FFNx-Steam-v1.8.1.0. The game is fully working now!

Edit: FFNx-v1.13.0.0 is working.
